content/docs/command-reference/cache/dir.md

```diff
@@ -5,23 +5,34 @@ using `dvc config cache`).
 
 ## Synopsis
 
-```usage
+````usage
 usage: dvc cache dir [-h] [--global] [--system] [--local] [-u] value
 
 positional arguments:
   value        Path to cache directory. Relative paths are resolved
-               relative to the current directory and saved to config
-               relative to the config file location.
-```
+               relative to the current directory and saved to config# cache dir
+
+Set/unset the <abbr>cache</abbr> directory location intuitively (compared to
+using `dvc config cache`).
+
+## Synopsis
+
+```usage
+usage: dvc cache dir [-h] [--global] [--system] [--local] [-u] value
```

Comment:
This looks like an error. The code block should only have 3 back quotes, not nested, and the cmd definition as well as "Synopsis" H2 header are repeated. Please revert.